__test__/creds/VerifiableCredential.test.js

```diff
@@ -1834,33 +1830,118 @@ describe('Transient Credential Tests', () => {
 describe('Signed Verifiable Credentials', () => {
   beforeAll(() => {
     schemaLoader.addLoader(new CVCSchemaLoader());
+
+    didTestUtil.mockDids();
   });
 
   beforeEach(() => {
     schemaLoader.reset();
   });
 
   test('Should create a verifiable credential instance', async () => {
+    const verificationMethod = `${didTestUtil.DID_SPARSE}#default`;
+    const keypair = didTestUtil.keyPair(didTestUtil.DID_SPARSE);
+
     const name = await Claim.create('claim-cvc:Identity.name-v1', identityName);
     const dob = await Claim.create('claim-cvc:Identity.dateOfBirth-v1', identityDateOfBirth);
-    const cred = await VC.create('credential-cvc:Identity-v3', uuidv4(), null, credentialSubject, [name, dob], null,
-      new CredentialSignerVerifier({ prvBase58 }));
+
+    const cred = await VC.create(
+      'credential-cvc:Identity-v3',
+      didTestUtil.DID_SPARSE,
+      null,
+      credentialSubject,
+      [name, dob],
+      null,
+      {
+        verificationMethod,
+        keypair,
+      },
+    );
+
     expect(cred).toBeDefined();
-    expect(cred.proof.merkleRootSignature).toBeDefined();
-    expect(cred.verifyMerkletreeSignature(pubBase58)).toBeTruthy();
+    expect(cred.proof.merkleRootSignature.signature).toBeDefined();
+    expect(cred.proof.merkleRootSignature.verificationMethod).toBe(verificationMethod);
+
+    // TODO: re-activate this once verify is done
```

Comment:
is there a ticket for this? If so please add the number here so we know when to remove the todo